Mathlib/Algebra/Ring/CentroidHom.lean

```diff
@@ -13,7 +13,7 @@ import Mathlib.Algebra.Ring.Subsemiring.Basic
 /-!
 # Centroid homomorphisms
 
-Let `A` be a (non unital, non associative) algebra. The centroid of `A` is the set of linear maps
+Let `A` be a (non unital, nonassociative) algebra. The centroid of `A` is the set of linear maps
```

Comment:
Also here, for the hyphen

Reply:
See e.g.

https://ncatlab.org/nlab/show/nonassociative+group

And CMOS.

Reply:
Changed to non-associative based on usage stats.